playwright.config.ts

```diff
@@ -7,19 +7,25 @@ if (process.env.CI) {
 }
 
 const config: PlaywrightTestConfig = {
-  // retries: 0, // default
+  testDir: './tests',
+  retries: 2, // default
   // workers: 50%, // default
-  testMatch: '*/**/e2e.ts',
+  testMatch: '*/**/e2e/*.ts',
```

Comment:
Maybe we could go to `'*/**/*e2e.ts',`
To have `page.e2e.ts` for example.

I'm not sure `/e2e.ts` would be matching ?!

To be checked